src/components/transfers/custom/FormSelect.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useField, useFormikContext } from 'formik'
+import { useTranslation } from 'next-i18next'
+
+import { TextField, TextFieldProps } from '@mui/material'
+
+import { translateError } from 'common/form/useForm'
+import { TranslatableField } from 'common/form/validation'
+
+export type selectProps = {
+  label: string
+  name: string
+} & TextFieldProps
+
+export default function FormSelect({ label, name, ...textFieldProps }: selectProps) {
+  const { t } = useTranslation()
+  const [field, meta] = useField(name)
+  const { setFieldValue } = useFormikContext()
+
+  const helperText = meta.touched ? translateError(meta.error as TranslatableField, t) : ''
+
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setFieldValue(name, event.target.value)
+  }
+
+  textFieldProps.size = 'small'
+  textFieldProps.fullWidth = true
+  textFieldProps.variant = 'outlined'
```

Comment:
You can define this on the component level instead of overriding them like that:

When the spreading os the last argument it overwrites them

```tsx
<TextField
  fullWidth
  size="small"
  variant="outlined"
  {...textFieldProps}
/>
```